src/ipinfo.rs

```diff
@@ -321,6 +321,40 @@ impl IpInfo {
         Ok(details)
     }
 
+    /// Get a mapping of a list of IPs on a world map
+    ///
+    /// # Example
+    ///
+    /// ```no_run
+    /// use ipinfo::IpInfo;
+    /// 
+    ///  #[tokio::main]
+    /// async fn main() {
+    ///     let map_url = IpInfo::get_map(&["8.8.8.8", "4.2.2.4"]).await.expect("should run");;
+    /// }
+    /// ```
+    pub async fn get_map(ips: &[&str]) -> Result<String, IpError> {
+        if ips.len() > 500_000 {
+            return Err(err!(MapLimitError));
+        }
+
+        let map_url = "https://ipinfo.io/tools/map?cli=1";
```

Comment:
```suggestion
        let map_url = &format!("{}/tools/map?cli=1", self.url);
```

I think we should make the `get_map` function an instance method to be consistent with rest of the API functions (`lookup`) instead of a static function. Would also be easier to understand from consumer point of view. 
But yes, making it static would also do as it doesn't right now use any other instance information that `url`.

Reply:
It's actually a totally separate function and we don't wanna initialize the cache or other default configuration as we don't even need token for generating the map the only thing required is the client. 

We're following the same static approach in other SDKs as well like go, java and node.

Reply:
I think it's fine going with this approach. @UmanShahzad you got any input on this?

Reply:
In Java is it a static method or on the class instantiation? We should follow what Java did in this case.

I think in most languages though like Python and Nodejs we did it on the class.

Reply:
Just checked and actually in Java, node, and python it's a class instantiation. Ya, we can go with the same approach in the Rust as well then. I was just thinking that it's useless and resource-consuming to load all the country info related files and cache especially when we're not gonna use it.

Reply:
We discussed it in person

Reply:
rust doesn't support function overloading hence we can't go with the wrapper function approach, so we have to make a small sacrifice on performance for a better UX.